src/QMCTools/PyscfToQmcpack_Spline.py

```diff
@@ -0,0 +1,936 @@
+######################################################################################
+## This file is distributed under the University of Illinois/NCSA Open Source License.
+## See LICENSE file in top directory for details.
+##
+## Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+##
+## File developed by: Yubo "Paul" Yang, yubo.paul.yang@gmail.com, University of Illinois Urbana-Champaign
+##                    Thomas Applencourt, applencourt@anl.gov,  Argonne National Laboratory
+##                    Hyeondeok Shin, hshin@anl.gov, Argonne National Laboratory          
+##                    Anouar Benali, benali@anl.gov, Argonne National Laboratory
+##                  
+## File created by: Thomas Applencourt, applencourt@anl.gov, Argonne National Laboratory
+#######################################################################################
+
+import numpy as np
+import h5py
+import os
+try:
+  from lxml import etree
+except:
+  import sys
+  sys.exit("Error: lxml module is needed for the generation of the XML file. Please install it using your favorite package manager")
+
+try:
+  import pandas as pd
+except:
+  import sys
+  sys.exit("Error: Panda module is needed for the save_eigensystem and eigensystem functions. Please install it using your package manager")
+
+
+def pyscf2qmcpackspline(cell,mf,title="Default", kpts=[], kmesh=[],  sp_twist=[]):
+  import sys, re
+
+  Restricted=True
+  PBC=False
+  Gamma=False
+
+  # Python version check
+  if sys.version_info < (3, 2):
```

Comment:
When loading this file, Python 2 gives a syntax error on the f-strings (around line 75).  This version check will never be executed.   f-strings were introduced in Python 3.6, so presumably the same thing will happen for python 3 versions 3.5 and lower (but I didn't check, only ran under Python 2).

Reply:
I can fix the "fstring" issue. 
Do you have a minimun python version for QMCPACK? 
I can either remove all the "fstring" (I will be tedious but I understand) or just fix the warning.